modules/oxide/src/less/skins/content/dark/content.less

```diff
@@ -4,17 +4,22 @@
  * For LGPL see License.txt in the project root for license information.
  * For commercial licenses see https://www.tiny.cloud/
  */
+@import 'src/less/theme/globals/global-variables.less';
```

Comment:
I agree with @lnewson  here. It seems through that @noxuhax only uses the break point variable. Instead of pulling in the global variables file here, just hard-code the breakpoint in the media query.